syntax = "proto3";

package MeterConnectorProto;

// The greeting service definition.
service MeterConnector {
  // Sends a greeting
  rpc readMeter (SMDataRequest) returns (SMDataReply) {}
}

// The request message containing the user's name.
message SMDataRequest {

}

// The response message containing the greetings
message SMDataReply {
  string message = 1;
}